xrmocap/transform/image/color.py

```diff
@@ -66,8 +87,40 @@ def bgr2rgb(input_array: Union[np.ndarray, torch.Tensor],
     b_slice_list[color_dim] = slice(2, 3, 1)
     if isinstance(input_array, torch.Tensor):
         b_backup = input_array[tuple(b_slice_list)].clone()
+        if not inplace:
+            ret_array = input_array.clone()
+        else:
+            ret_array = input_array
     else:
         b_backup = input_array[tuple(b_slice_list)].copy()
-    input_array[tuple(b_slice_list)] = input_array[tuple(r_slice_list)]
-    input_array[tuple(r_slice_list)] = b_backup
-    return input_array
+        if not inplace:
+            ret_array = input_array.copy()
+        else:
+            ret_array = input_array
+    ret_array[tuple(b_slice_list)] = input_array[tuple(r_slice_list)]
+    ret_array[tuple(r_slice_list)] = b_backup
```

Comment:
Save `tuple(b_slice_list)` and `tuple(r_slice_list)` to two variables in adavance? It seems that they have been used serveral times.

Reply:
Fixed in [db0b34d](https://github.com/openxrlab/xrmocap/pull/78/commits/db0b34dfe3f5b5f8de5992a88bff7b99c5cd21ea).